src/test/ui/resolve/issue-21221-1.stderr

```diff
@@ -16,10 +18,11 @@ error[E0412]: cannot find type `Mul` in this scope
    |                ^^^ not found in this scope
    |
 help: possible candidates are found in other modules, you can import them into scope
-   | use mul1::Mul;
-   | use mul2::Mul;
-   | use mul3::Mul;
-   | use mul4::Mul;
+   |
+11 | use mul1::Mul;
+11 | use mul2::Mul;
+11 | use mul3::Mul;
+11 | use mul4::Mul;
```

Comment:
The underline only happens if there's only one suggestion that is on a single line (didn't want to have fancy multiline markings in the suggestions).

A multiline span suggestion looks the following way at the moment:

```
error: `<` is interpreted as a start of generic arguments for `usize`, not a comparison
  --> ../../src/test/ui/issue-22644.rs:20:14
   |
19 |              <
   |              - not interpreted as comparison
20 |              b);
   |              ^ interpreted as generic argument
   |
help: if you want to compare the casted value then write:
   |
16 |     println!("{}", (a
17 |              as
18 |              usize)
   |
```

Two other possible ways to present multiple candidates are

```

error[E0412]: cannot find type `Mul` in this scope
  --> $DIR/issue-21221-1.rs:72:16
   |
72 | fn getMul() -> Mul {
   |                ^^^ not found in this scope
   |
help: possible candidates are found in other modules, you can import them into scope
   |
11 | use mul1::Mul;
   |
11 | use mul2::Mul;
   |
11 | use mul3::Mul;
   |
11 | use mul4::Mul;
   |
and 2 other candidates
```

and

```

error[E0412]: cannot find type `Mul` in this scope
  --> $DIR/issue-21221-1.rs:72:16
   |
72 | fn getMul() -> Mul {
   |                ^^^ not found in this scope
   |
help: possible candidates are found in other modules, you can import them into scope
   |
11 | use mul1::Mul;
   | ^^^^^^^^^^^^^^
11 | use mul2::Mul;
   | ^^^^^^^^^^^^^^
11 | use mul3::Mul;
   | ^^^^^^^^^^^^^^
11 | use mul4::Mul;
   | ^^^^^^^^^^^^^^
and 2 other candidates
```

---

Multiple suggestions that span multiple lines would look terrible with the current state of this PR:

```
error[E0412]: cannot find type `Mul` in this scope
  --> $DIR/issue-21221-1.rs:72:16
   |
72 | fn getMul() -> Mul {
   |                ^^^ not found in this scope
   |
help: possible candidates are found in other modules, you can import them into scope
   |
11 | use
12 | mul1::Mul;
11 | use
12 | mul2::Mul;
11 | use
12 | mul3::Mul;
11 | use
12 | mul4::Mul;
and 2 other candidates
```